workshop/Lab0/README.md

```diff
@@ -0,0 +1,114 @@
+# Lab 0: Install Prequisites
+
+Before we begin, we need to install Service Catalog and a few other things
+we'll need. Towards that end, this lab contains instructions to install:
+
+* The Helm CLI, version 2.7.0 or later
+* The Helm tiller pod inside your cluster
+* Service Catalog, version 0.1.18 or later
+* svcat, the Service Catalog CLI tool
+* create a PersistantVolume on our cluster
+
+# Set KUBECONFIG
+
+Make sure your KUBECONFIG environment variable is set to the output of
+
+`bx cs cluster-config [cluster-name]`
+
+or otherwise targeting your IBM Cloud Kubernetes Service cluster.
+
+# Install Helm
+
+1. Download and install [the Helm CLI](https://github.com/kubernetes/helm#install).
+2. Run `helm init` to install the tiller pod into your kube cluster. This pod is Helm's entry point
+for creating additional infrastructure inside your cluster.
+3. To allow the tiller pod permissions to perform changes inside your cluster, run
+
+```
+kubectl create clusterrolebinding tiller-cluster-admin \
+--clusterrole=cluster-admin \
+--serviceaccount=kube-system:default
+```
+
+# Install Service Catalog
+
+Service Catalog is installed via a Helm chart, a manifest that describes what to install and how to run it.
```

Comment:
Feels a bit odd to do this here instead of right before we use it. I would consider moving it to a future lab - or if you do keep it here add some text about how we won't use it until later, but we're just having them pre-install everything in advance.

Reply:
Is the boilerplate text at the top of Lab0 not sufficient for that? A lot of stuff that I tell them to install in Lab0 isn't needed right away.

Reply:
yeah, I guess its ok